README.md

```diff
@@ -1,115 +1,40 @@
-# plugin-&lt;REPLACE ME&gt;
+# Salesforce DX
 
-Change above to <REPLACE_ME> before finalizing
+A core plugin to the Salesforce DX CLI that provides all commands under the force topic.
```

Comment:
Remove DX. It is just the Salesforce CLI.

Also, maybe say something like:

The Salesforce DX plugin aggregates all `force` topic commands into a single plugin that is bundled with the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli).